scss/inc/rtl/_action-bar.scss

```diff
@@ -76,5 +76,8 @@
 
       }
     }
+    .icon-external{
+      transform: scale(-1, 1);
```

Comment:
It is flipped, but does it need padding change as well?
![Screenshot 2021-11-30 at 15 24 03](https://user-images.githubusercontent.com/43652944/144065018-8717166a-0624-49ed-b4b7-11c737bcc19e.png)
![Screenshot 2021-11-30 at 15 24 13](https://user-images.githubusercontent.com/43652944/144065030-b8c40cad-b994-4243-9f03-9e9507a18b3a.png)

Look at the SCSS above, all the other icons were done in a block with padding updated.

Reply:
Good catch